src/instructlab/model/train.py

```diff
@@ -282,6 +282,12 @@ def clickpath_setup(is_dir: bool) -> click.Path:
     is_flag=True,
     help="if true, enables the legacy linux training code path from release 0.17.0 and prior.",
 )
+@click.option(
+    "--disable-flash-attn",
+    is_flag=True,
+    help="If true, this disables the use of flash attention, resulting in slower training. This is useful on older systems where flash-attention isn't supported.",
```

Comment:
the default and the help can come from the configuration

Reply:
Good catch, I'll get rid of this